### DISCUSSION TITLE
Keybinding next / previous window

### DISCUSSION THREAD

**Discussion by peterpeterparker** [zed-industries/zed#12230] [Feature Requests]
Hi 👋,

Is there a keybinding I could use to set up a custom shortcut to switch between Zed windows? 

For example, if I have mutiple projects open, I would like to switch between those using a custom shortcut.

```
[
  {
    "context": "Editor",
    "bindings": {
      "cmd-§": "??????" // <--- Switch to next window / project open in Zed as well
    }
  }
]
```

**Comment by SomeoneToIgnore** [comment]
Zed's windows are native window manager ones, so whatever related OS shortcut should be used.
macOS typically uses cmd-` for that.

**Reply by peterpeterparker** [reply]
Thanks, works out!

If anyone ever needs it, I used this article to find out how to set up a custom shortcut on Mac: https://techwiser.com/ways-to-switch-between-windows-of-same-app-in-mac/

**Comment by jamarju** [comment]
This seems broken on macOS with a Spanish keyboard layout. On this layout, \` is a dead key.

cmd-\` does move to the next window generally, but in Zed the backtick key is captured and inserted into the current file instead.

The same shortcut works correctly for me in Chrome and iTerm2, among others, so this looks Zed-specific.

**Reply by yannickschuchmann** [reply]
For me (german keyboard layout) the switching with backtick is not working while I have a file open and focused. Once I click in the file tree sidebar to focus a file there, the backtick shortcut works.

**Reply by LRDN** [reply]
You can unbind or change the conflicting default key binding.
```
{
  "context": "Editor",
  "unbind": {
    "cmd-`": "editor::ExpandAllDiffHunks"
  }
}
```

**Reply by yannickschuchmann** [reply]
life changing! thx, @LRDN !
